larray/viewer/api.py

```diff
@@ -155,7 +155,26 @@ def view(obj=None, title='', depth=0):
 
 
 def compare(*args, **kwargs):
+    """
```

Comment:
That's nice indeed, but that cannot work in all cases (spaces or special character in names etc.) so that cannot be the only way (and I am a bit tired of our habit to provide 6 different ways to do the same thing).

maybe a compromise would be compare({'a1': a1, 'a2': a2})